actions/DisplayAction.php

```diff
@@ -224,7 +224,11 @@ public function execute() {
 
 						$items[] = $item;
 					} elseif(Configuration::getConfig('error', 'output') === 'http') {
-						header('Content-Type: text/html', true, $this->get_return_code($e));
+						$code = $this->get_return_code($e);
+						if ($code == 503) {
+							header('Retry-After: 600', true);
+						}
+						header('Content-Type: text/html', true, $code);
```

Comment:
Could you explain, why do we need it here? And why 600 (10 minutes)?

Reply:
I'm trying to tell the RSS reader that the error is temporary.

All the Twitter 403 errors I've received over the last few weeks have been temporary, so I wanted a way to tell the RSS reader to try again soon.  I don't know if any readers treat 500 different from 503.

The 10 minutes is just a guess.